packages/plugin-azuremaps/src/OmhMapsPluginAzureMapsModule.ios.ts

```diff
@@ -0,0 +1,6 @@
+import { Spec } from './NativeOmhMapsPluginAzureMapsModule';
+
+// Note: iOS implementation is not available
```

Comment:
We need to provide `stubs`, otherwise iOS app crashes whenever we want to call some `setSubscriptionKey` in our application.

In sample app it doesn't crash as we set those keys natively.